okhttp/src/main/java/okhttp3/internal/ws/RealWebSocket.kt

```diff
@@ -81,6 +83,12 @@ class RealWebSocket(
   /** Names this web socket for observability and debugging. */
   private var name: String? = null
 
+  /**
+   * Null until web socket is connected with compression enabled.
+   * Used for compressing data at message level.
+   */
+  private var messageDeflater: MessageDeflater? = null
```

Comment:
I think it’s weird that this is on the RealWebSocket but the inflater is on the MessageReader. Seems needlessly asymmetric.